grafana-plugin/src/utils/authorization/index.ts

```diff
@@ -90,12 +90,30 @@ export const userHasMinimumRequiredRole = (minimumRoleRequired: OrgRole): boolea
  *
  * As a fallback (second argument), for cases where RBAC is not enabled for a grafana instance, rely on basic roles
  */
-export const isUserActionAllowed = ({ permission, fallbackMinimumRoleRequired }: UserAction): boolean => {
-  if (config.featureToggles.accessControlOnCall) {
-    return !!contextSrv.user.permissions?.[permission];
-  }
-  return userHasMinimumRequiredRole(fallbackMinimumRoleRequired);
-};
+export const isUserActionAllowed = ({ permission, fallbackMinimumRoleRequired }: UserAction): boolean =>
+  config.featureToggles.accessControlOnCall
+    ? !!contextSrv.user.permissions?.[permission]
+    : userHasMinimumRequiredRole(fallbackMinimumRoleRequired);
+
+/**
+ * Given a `UserAction`, returns the permission or fallback-role, prefixed with "permission" or "role" respectively
+ * depending on whether or not RBAC is enabled/disabled
+ *
+ * @param UserAction
```

Comment:
We're not generating anything with JSDoc at the moment and we do not have any code documented with `@param` or `@returns` - maybe it'd make sense to remove these

Reply:
addressed [here](https://github.com/grafana/oncall/pull/1185/commits/48cf7c3de02afd763f1fdf05cc0b9177c76eccaa) 👍